vcsim/main.go

```diff
@@ -63,6 +63,7 @@ func main() {
 	cert := flag.String("tlscert", "", "Path to TLS certificate file")
 	key := flag.String("tlskey", "", "Path to TLS key file")
 	env := flag.String("E", "-", "Output vcsim variables to the given fifo or stdout")
+	listen := flag.String("l", "127.0.0.1:8989", "Listen address for vcsim")
```

Comment:
How does listen differ from httptest.serve?

Reply:
It's just an alias, they behave the same.  But @lamw had suggested a while back that `httptest.serve` wasn't obvious.  Decided to address that while I was at it.